<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingCam</class>
 <widget class="QMainWindow" name="SettingCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки камеры</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Фото лица</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="face_list_cam"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="cam_photo">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="chacked_cam_face">
           <property name="text">
            <string>Проверить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_cam_face">
           <property name="text">
            <string>Остановить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Фото паспорта</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QComboBox" name="pasport_list_cam"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="cam_pasport">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="checked_cam_pasport">
           <property name="text">
            <string>Проверить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_cam_pasport">
           <property name="text">
            <string>Остановить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="save_setting_cam">
      <property name="text">
       <string>Сохранить настройки</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
